eng/native/configurecet.cmake

```diff
@@ -0,0 +1,9 @@
+# Enable EH-continuation table and CET-compatibility for native components for amd64 builds.
```

Comment:
It should still be included from the configurecompiler.cmake or rather moved back there (ideally co-located together like you have it here). All compiler options common for coreclr, libraries and installer should be there. We should just wrap it with an "if" that filters it out for Mono.

Reply:
Sounds good, done